scripts/test_mmlu.py

```diff
@@ -1,13 +1,19 @@
 # First Party
 from instructlab.eval.mmlu import MMLUEvaluator
 
+SYSTEM_PROMPT = """I am, Red Hat® Instruct Model based on Granite 7B, an AI language model developed by Red Hat and IBM Research, based on the Granite-7b-base language model. My primary function is to be a chat assistant."""
+
 
 def test_minimal_mmlu():
     print("===> Executing 'test_minimal_mmlu'...")
     try:
         model_path = "instructlab/granite-7b-lab"
         tasks = ["mmlu_anatomy", "mmlu_astronomy"]
```

Comment:
In this test should we also check the template is applied. I think the evaluator also return logs. Maybe we can check them?

Reply:
Yes, that's a good test to add.

Reply:
Added this test

Reply:
Thanks. This looks good